mlflow/sklearn.py

```diff
@@ -64,8 +64,8 @@ def _load_model_from_local_file(path):
         return pickle.load(f)
 
 
-def load_pyfunc(path):
-    with open(path, "rb") as f:
+def load_pyfunc(data_path):
```

Comment:
Q: why change this API?

Reply:
This was a leftover of an earlier version. I reverted it back to `path`.